README.adoc

```diff
@@ -35,6 +35,8 @@ Further elements will be supported in the future. Feel free to open issues https
 == Installation
 
     $ go get -u github.com/bytesparadise/libasciidoc
+    $ make install-devtools
+    $ make install
```

Comment:
in that case, I would suggest that the `install-devtools` becomes a dependency on the `install` goal, so the user doesn't need to call `make install-devtools` explicitly.

Reply:
Done

Reply:
awseome! Thanks for this PR, @moorereason!